repo-agent/pkg/llm/command_expansion.go

```diff
@@ -0,0 +1,96 @@
+package llm
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// Regex for finding commands: /namespace:command "args" or /namespace:command args
+// We'll support quoted args with simple quote handling.
+var commandRegex = regexp.MustCompile(`(?m)/([\w-]+):([\w-]+)(?:\s+(".*?"|[^\s]+))?`)
+
+func expandCommands(prompt string, geminiDir string) (string, error) {
+	return replaceCommands(prompt, geminiDir)
+}
+
+func replaceCommands(text string, geminiDir string) (string, error) {
+	// We iterate through all matches and replace them.
```

Comment:
The regex only captures arguments in double quotes. It does not support single quotes or unquoted arguments. For better usability, consider extending the regex to handle these cases, especially for simple, single-word arguments.